test/features/datarequest.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Datarequest
+
+# Data Requests - Install and configure extensionUAT - the Data Requests are accessible via the /datarequest URL (does not display a 404 error)
+    Scenario: Browse to data request page
```

Comment:
comments needs to be added to scenario name. This needs to be done for all of them. 
Just add a - and append to the end.

i.e. 
Browse to data request page
 becomes
Browse to data request page - Data Requests - Install and configure extensionUAT - the Data Requests are accessible via the /datarequest URL (does not display a 404 error)

Please do to all of them